aquacomputer_d5next.c

```diff
@@ -1642,6 +1648,8 @@ static int aqc_write(struct device *dev, enum hwmon_sensor_types type, u32 attr,
 			case aquaero:
 				switch (val) {
 				case 0:
+					ctrl_mode = AQUAERO_CTRL_PRESET_ID_AUTO;
```

Comment:
What is "real automatic control mode" referring to in aquasuite? Not sure I understand.

Reply:
I'm referring to mode 0 as described here, so automatic control mode seems not to be worded ideally.
> pwm[1-*]_enable
		Fan speed control method:
		0: no fan speed control (i.e. fan at full speed)
		1: manual fan speed control enabled (using pwm[1-*])
		2+: automatic fan speed control enabled
		Check individual chip documentation files for automatic mode
		details.
		RW
And I'm saying real as we previously faked this mode by just setting it to pwm and pwm to 255.

Reply:
I should probably rename it to `AQUAERO_CTRL_PRESET_ID_NO_CONTROL`

Reply:
Understood, thanks.